src/docs/deployment/flavors.md

```diff
@@ -6,12 +6,19 @@ description: How to create build flavors specific to different release types or
 
 Do you need to set up product flavors for different development
 environments or release types?
-The community has written some articles you might find useful.
-These articles address flavors for both iOS and Android.
+The community has created some packages that you might find useful.
 
-See [Creating flavors of a Flutter app][], [Flavoring Flutter][],
-and [Flutter Ready to Go][].
 
-[Creating flavors of a Flutter app]: https://cogitas.net/creating-flavors-of-a-flutter-app/
-[Flavoring Flutter]: {{site.medium}}/@salvatoregiordanoo/flavoring-flutter-392aaa875f36
-[Flutter Ready to Go]: {{site.medium}}/flutter-community/flutter-ready-to-go-e59873f9d7de
+The following packages are listed alphabetically:
+
+* [`flutter_flavor`][]
+* [`flutter_flavorizr`][]
+* [`kotlin_flavor`][]
```

Comment:
@sfshaza2 

I think https://pub.dev/packages/kotlin_flavor isn’t related to the flavor for different development
 environments or release types 🤔